build/yamls/antrea-aks.yml

```diff
@@ -3911,9 +3911,20 @@ data:
     #tlsMinVersion:
 
     # The name of the interface on Node which is used for tunneling or routing the traffic across Nodes.
-    # If there are multiple IP addresses configured on the interface, the first one is used.
-    # The interface configured with Node IP is used if this parameter is not set.
+    # If there are multiple IP addresses configured on the interface, the first one is used. The order for
+    # configuring tunneling or routing the traffic across Nodes is (from highest to lowest):
+    # 1. transportInterface
+    # 2. transportAddressCIDR
+    # 3. The Node IP
     #transportInterface:
```

Comment:
@tnqn @jianjuns How about rename `transportInterface` to `transportInterfaceName`, it will ba easy to distinguish between `transportInterfaceName` and `transportInterfaceCIDRs`.

Reply:
I agree transportInterfaceName sounds better, but also feel it does not matter much. Probably not worthwhile to rename a parameter.

Reply:
> I agree transportInterfaceName sounds better, but also feel it does not matter much. Probably not worthwhile to rename a parameter.

Yes, maybe transportInterface parameter has been used in some k8s clusters, rename it may cause some unnecessary trouble.